src/main/kotlin/no/nav/familie/ef/sak/api/gui/SøkController.kt

```diff
@@ -21,4 +25,9 @@ class SøkController(private val søkService: SøkService, private val tilgangSe
         tilgangService.validerTilgangTilPersonMedBarn(personIdentRequest.personIdent)
         return Ressurs.success(søkService.søkPerson(personIdentRequest.personIdent))
     }
+
+    @PostMapping("person/adresse")
```

Comment:
Har noen forslag på disse?
Vi har en controller som er "søk" med en post direkt på `api/sok`, og så denne.

Reply:
`person/samme-adresse` er kanskje litt langt, men mer informativt? 

Ellers - vi trenger ikke noe tilgangskontroll på denne, eller?

Reply:
hvilken adresse saksbehandler har mulighet til å se håndteres av pdl. --->"Søk på geografiske opplysninger føre til til at personer med adressebeskyttelse vil bli filtrert vekk fra søke resultatene, det gjøres unntak for saksbehandlere som tilgang til å se personer med adressebeskyttelse."

Reply:
Vi sender med azure-token til PDL for denne, så att PDL håndterer tilgangskontroll :)